algorithms/ar-lis/javascript/longest_increasing_subsequence.js

```diff
@@ -0,0 +1,30 @@
+/**
+ * Longest increasing subsequence is the subsequence that has the longest streak of going up consecutively
+ * @param {number[]} arr the sequence to search for longest subsequence
+ */
+function findLongestIncreasingSubsequence(arr) {
+    let longestSubsequence = [];
+    for (let arrKey in arr) {
+        if (Number(arrKey) + 2 > arr.length) break;
+        let tempLength = 0;
+        let isIncreasing = true;
+        while (isIncreasing) { // if next element is greater than current element, subsequence is increasing
+            if (arr[Number(arrKey) + tempLength] >= arr[++tempLength + Number(arrKey)])
```

Comment:
Codacy found an issue: [Avoid assignments in operands](https://app.codacy.com/gh/FOSS-UCSC/FOSSALGO/pullRequest?prid=6422328)